frontend/src/editor/index.tsx

```diff
@@ -159,8 +164,13 @@ export function EditorScreen({
 			</div>
 
 			<div
-				className={`overflow-y-scroll ${isDemo ? classes.demosidebar : classes.sidebar}`}
-			>
+  className={`overflow-y-scroll ${
+    isDemo
+      ? "m-5 h-[80vh] min-w-[28rem] w-[28rem] shadow-[ -2px_0_5px_rgba(0,0,0,0.1)] font-sans bg-white"
```

Comment:
eventually we'll want to restructure the layout and remove that `h-[80vh]`. But that's a separate change. This lgtm so far... once you pull in `classes.sidebar` for the fallback.